doc/comClassUML.txt

```diff
@@ -0,0 +1,41 @@
+# Copy and paste the following into plantuml.com
+
+@startuml
+class Config
+class ComServer
+class ComConnection
+class ComControl
+class ComTelemetry
+ComControl : theoretical, unimplemented
+ComTelemetry : theoretical, unimplemented
+
+Config .. ComServer
+ComServer *-- ComConnection
+ComConnection <|-- ComControl
+ComConnection <|-- ComTelemetry
+ComControl *-- NetCommandFactory
+ComTelemetry *-- NetCommandFactory
+
+class NetCommandException
+class NetCommandFactory
+NetCommandFactory : Create new NetCommand from one found in _cmdMap
+entity _cmdMap
+_cmdMap : <CommandName, NetCommand>
+abstract class NetCommand
+NetCommand : Children vary in action() and json parameters
+class NCmdAck
+class NCmdNoAck
+class NCmdEcho
+
+NetCommandException .. NetCommand : throws
```

Comment:
Please help to add the arrow for this dependency if it is single direction. Is it?
```
NetCommandException <.. NetCommand : throws
```

Reply:
It isn't really an "is a" or "has a" type of relationship. I guess throws is kind of like 'o..'.